anyway/backend_constants.py

```diff
@@ -120,3 +84,109 @@ class ResolutionCategories(Enum):
 
 
 BE_CONST = BackEndConstants()
+
+
+class LabeledCode(Enum):
+
+    def get_label(self) -> str:
+        return self.labels()[self]
+
+    @classmethod
+    def codes(cls: Iterable) -> List[int]:
+        if isinstance(cls, Iterable):
+            return [a.value for a in cls]
+        else:
+            raise ValueError(f'{cls}: needs to be derived from Enum')
+
+    def labels(self):
+        return {}
+
+
+# This is a type for the field 'injury_severity' in the table 'involved_markers_hebrew'
+class InjurySeverity(LabeledCode):
+    KILLED = 1
+    SEVERE_INJURED = 2
+    LIGHT_INJURED = 3
+
+    def labels(self):
```

Comment:
IMO this should be a `@classmethod` accepting `cls` instead of `self` (then you can write for example `cls.KILLED`, but you can also ignore the parameter and keep the implementation as it is). This can't be an instance method, since it has nothing to do with the specific instance.